packages/orchestration/src/orchestration-chat-completion-stream-chunk-response.ts

```diff
@@ -0,0 +1,45 @@
+import type { CompletionPostResponseStreaming, LLMChoiceStreaming } from './client/api/schema/index.js';
+import type { TokenUsage } from './index.js';
+
+/**
+ * Orchestration chat completion stream chunk response.
+ */
+export class OrchestrationChatCompletionStreamChunkResponse {
```

Comment:
Currently, this API is lacking the orchestration specific results, i.e. users can't access `module_results`

Reply:
Yeah I had a convenience method to get the different module_results, but wasn't sure if anyone actually would use them.

You can still access them though, just not through a function, instead it's `chunk.data.module_results?.YOUR_MODULE`

Reply:
What do you think, should there be an additional function for it?

Previously my method was `.getModuleResult(<select-module-with-auto-complete-here>)`

Reply:
I would assume the module results would be useful on the overall response object? And there it's currently not available, right?

![image](https://github.com/user-attachments/assets/e4eae016-ba91-4d08-a1ac-0b200cd0e70a)

I guess it depends a bit on the specific results one is interested in. If we take a chat bot as example use case then I need the templating result + full LLM response at the end and store it in memory. Currently, I'd have to aggregate this manually across the chunks, right?

Reply:
I see, so you would like to save all module_results in an array which is available on the response object?

Reply:
yeah, but I would expect we aggregate over all chunks, so there is only one module_results outcome at the end that includes everything

Reply:
I see, I suppose this needs a little bit of investigation to see which implementation (e.g. aggregation) would be the most suitable, I would suggest I create a follow-up ticket for this?

Reply:
follow up ticket is also fine 👍🏻